### PR TITLE
fix: Add `isIconStatic` prop item to allow the icons to be always rendered statically ignoring hover behaviour

### LINKED ISSUE #4653: View Switcher Lock Design
### Current Behavior

The lock is displayed on hover.

https://github.com/twentyhq/twenty/assets/19412894/15286239-4523-4592-8ec3-52b22158e9f6

### Desired Behavior

The lock should always be visible. Its color should be `Text/Tertiary`

CleanShot 2024-03-26 at 10 00 33

### PR DESCRIPTION
**Pull request by besrabasant**
Fix #4653

### CODE REVIEW

File: packages/twenty-front/src/modules/views/view-picker/components/ViewPickerListContent.tsx [hunk 1]
```diff
@@ -74,6 +74,7 @@ export const ViewPickerListContent = () => {
                     Icon: IconLock,
                   },
             ].filter(isDefined)}
+            isIconStatic={ view.key !== 'INDEX'}
```
Comment (bosiraphael) [inline packages/twenty-front/src/modules/views/view-picker/components/ViewPickerListContent.tsx:77]:
```suggestion
            isIconStatic={view.key !== 'INDEX'}
```

**Review by bosiraphael — COMMENTED** [review]
Hello ! Thank you for contributing :)
We don't really use inline conditional classNames in our project, we prefer to use a Styled component for this purpose. Could you create a Styled div which takes isIconStatic as an argument and returns a styled div with the correct className?
Could you also correct the lint issues?
Thank you :)

**Comment by bosiraphael** [comment]
@besrabasant I just saw that there was another PR opened before yours here: https://github.com/twentyhq/twenty/pull/4671 for the same issue.

**Comment by charlesBochet** [comment]
The other PR has actually been closed!

**Comment by bosiraphael** [comment]
Hello @besrabasant! Since the other PR has been closed, do you think you could make the changes I suggested above? Thank you :)

**Comment by FelixMalfait** [comment]
I will make the changes

**Comment by FelixMalfait** [comment]
Thanks for your help @besrabasant!

**Pull request merged**